sermant-plugins/sermant-router/dubbo3-router-plugin/src/test/java/com/huaweicloud/sermant/router/dubbo/interceptor/RegistryDirectoryInterceptorTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2024-2024 Huawei Technologies Co., Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.huaweicloud.sermant.router.dubbo.interceptor;
+
+import com.huaweicloud.sermant.core.plugin.agent.entity.ExecuteContext;
+import com.huaweicloud.sermant.router.common.cache.DubboCache;
+
+import org.apache.dubbo.common.URL;
+import org.apache.dubbo.rpc.RpcContext;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * dubbo3.x all/interface注册类型时设置应用名
+ *
+ * @author chengyouling
+ * @since 2024-03-18
+ */
+public class RegistryDirectoryInterceptorTest {
+    private static final String SERVICE_NAME = "test-service";
+
+    private static final URL APACHE_URL = URL.valueOf(
+            "dubbo://localhost:8081/com.huaweicloud.foo.FooTest?foo=bar&version=0.0.1&application=" + SERVICE_NAME);
+
+
+    private static final String SERVICE_INTERFACE = "com.huaweicloud.foo.FooTest";
```

Comment:
don't use com.huaweicloud. We will change all the package name. Modify all test cases

Reply:
how to modify, this step can do in the later stage？

Reply:
just use a custom demo package name in SERVICE_INTERFACE and APACHE_URL ，it does not matter

Reply:
fixed